hrui/hrui.php

```diff
@@ -399,24 +396,44 @@ function hrui_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
 }
 
 /**
- * Implementation of hook_civicrm_tabs
+ * Check if the extension with the given key is enabled
+ *
+ * @param string $extensionKey
+ * @return boolean
+ */
+function _hrui_check_extension($extensionKey)  {
+  return !!CRM_Core_DAO::getFieldValue(
```

Comment:
you can cast the value to boolean :

return (boolean) CRM_Core_DAO::getFieldValue ... etc 

it's more clear like that.